src/Tribe/Main.php

```diff
@@ -1361,8 +1377,8 @@ public function displayEventVenueDropdown( $postId ) {
 
 			?>
 			<tr class="">
-				<td style="width:170px"><?php printf( __('Use Saved %s:', 'tribe-events-calendar'), $this->singular_venue_label ); ?></td>
-				<td><?php $this->saved_venues_dropdown( $VenueID ); ?> <div class="edit-venue-link" <?php if( empty( $VenueID ) ) { ?>style="display:none;"<?php } ?>><a data-admin-url="<?php echo admin_url('post.php?action=edit&post=' ); ?>" href="<?php echo admin_url( sprintf( 'post.php?action=edit&post=%s', $VenueID ) ); ?>" target="_blank"><?php printf( __( 'Edit %s', 'tribe-events-calendar' ), $this->singular_venue_label ); ?></a></div></td>
+				<td style="width:170px"><?php echo esc_html( sprintf( __( 'Use Saved %s:', 'tribe-events-calendar' ), $this->singular_venue_label ) ); ?></td>
```

Comment:
It does not cause a bug in the way that you were thinking..._however_ it _may_ cause a bug if that value has been filtered by an end user and has had HTML added via the `tribe_venue_label_singular` filter.